entrypoint.sh

```diff
@@ -57,6 +56,21 @@ fi
 
 params="$params $tests"
 
-echo running: okteto "$command" "$params"
+if [ ! -z "$log_level" ]; then
+  if [ "$log_level" = "debug" ] || [ "$log_level" = "info" ] || [ "$log_level" = "warn" ] || [ "$log_level" = "error" ] ; then
```

Comment:
Nope, the okteto command will execute with the default one (`warn`), which might be another option, just let the CLI handle it. But in the rest of actions we are also validating it. So, if we remove the validation, I'd in the rest of actions too to be consistent

Reply:
Yeah yeah, agree, it is weird, it makes sense to delete it and don't validate at action level. Let's remove it that in the rest of actions too

Reply:
The validation was set after a suggestion on the deploy-preview https://github.com/okteto/deploy-preview/pull/53#discussion_r1611702433 

For me it makes sense to validate as we just accept certain values when using the action, the user should know if they are typing wrong the word, this would be specifically used when the user wants certain log level.
 if we want to pass whatever value is used because the CLI will validate (use default warn if is invalid) it might have a typo  an warn will be used, and user wont be aware of this.

I would prefer to leave it as is, but please, confirm to me it this should be changed and then i will change it in all the actions again.

Reply:
In that comment I didn't mean to add an explicit validation in the action but it is confusing and might look like it. And after that, I didn't say anything because I don't have strong opinion and I think both ways are fine. It is not something critical and both options have advantages and disadvantages

> For me it makes sense to validate as we just accept certain values when using the action, the user should know if they are typing wrong the word, this would be specifically used when the user wants certain log level.
if we want to pass whatever value is used because the CLI will validate (use default warn if is invalid) it might have a typo an warn will be used, and user wont be aware of this.

The same would happen when using directly the CLI as it is not validating anything.

Let's simplify things in the action and remove the log level, the less custom code in the github action the better.

Reply:
👍🏻  ok, i will open additional PRs alonside this change for the other actions. 
thanks for the feedback